spec/mailers/previews/share_obituary_preview.rb

```diff
@@ -0,0 +1,4 @@
+# Preview all emails at http://localhost:3000/rails/mailers/share_obituary
+class ShareObituaryPreview < ActionMailer::Preview
```

Comment:
These last two files were added in automatically, not sure if we need them at all.